schemas/PublicKeyRSA.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "PublicKeyRSA",
+  "description": "The public key for an RSA JSON Cleartext Signature",
+  "type": "object",
+  "properties": {
+    "type": {
+      "description": "RSA key type indicator.",
+      "type": "string",
+      "enum": [
+        "RSA"
+      ]
+    },
+    "n": {
```

Comment:
Would it be better to go with "modulus" and "exponent" and more descriptive names for "x" and "y" too?